internal/tools/orchestrator/scheduler/executor/plugins/k8s/terminal.go

```diff
@@ -212,7 +212,7 @@ func (k *Kubernetes) Terminal(namespace, podname, containername string, upperCon
 				return
 			}
 			m[0] = Output
-			if conf.TerminalSecurity() {
+			if conf.TerminalSecurity() || conf.TerminalMasking() {
```

Comment:
I think it should be `&&`.

Reply:
TerminalSecurity or TerminalMasking.
TerminalSecurity will deprecated, see [6474](https://github.com/erda-project/erda/pull/6474) pls.